.github/scripts/trymerge.py

```diff
@@ -1956,18 +1948,6 @@ def merge(
         dry_run=dry_run,
     )
 
-    if pr.last_pushed_at() is None:
-        print(
-            f"Can't get commit {pr.last_commit()['oid']} pushed date. Is it merge commit by chance?"
-        )
-    elif (datetime.utcnow() - cast(datetime, pr.last_pushed_at())).days > stale_pr_days:
```

Comment:
is there an equivalent for pusheddate that we can use? or are we fine with getting rid of the 3 day check for old jobs?

Reply:
The deprecation notice doesn't suggest any alternatives, and a cursory glance at the API didn't suggest anything either.

Frankly, this felt like a weird check that didn't really help with anything. The team seems to have originally thought this was a merge-base date check instead of a "last push" date check, the latter of which doesn't really offer much protection.

I do think we should add in a merge-base date check, but that's a separate project from getting rid of this deprecated code.  For the past month this code has been doing nothing anyways (I only discovered the failure because the test started failing once I regenerated the github gql query cache)